packages/editor/src/components/post-template/panel.js

```diff
@@ -65,19 +63,11 @@ export default function PostTemplatePanel() {
 	}, [] );
 
 	if ( ( ! isBlockTheme || ! canViewTemplates ) && isVisible ) {
-		return (
-			<PostPanelRow label={ __( 'Template' ) }>
-				<ClassicThemeControl />
-			</PostPanelRow>
-		);
+		return <ClassicThemeControl />;
```

Comment:
Perhaps by using a common `PostPanelRow` for both the classic theme and the block theme, `popoverProps` can be made common.

However, the conditions here are a bit complicated, so for now I've used the `PostPanelRow` component for both the classic theme control and the block theme control. Also, the conditions here may change as a result of the discussion in #64341.